src/components/proposals/proposal-card.vue

```diff
@@ -67,15 +67,23 @@ export default {
       return ''
     },
     background () {
-      if (this.voteConfig === null) return 'white'
-      const { vote } = this.voteConfig
-      if (vote === 'pass') return 'positive'
-      if (vote === 'abstain') return 'grey'
-      if (vote === 'fail') return 'negative'
+      if (this.$router.currentRoute.name === 'proposal-history') {
+        if (this.isAccepted) {
+          return 'positive'
+        } else {
+          return 'negative'
+        }
+      } else {
+        if (this.voteConfig === null) return 'white'
+        const { vote } = this.voteConfig
+        if (vote === 'pass') return 'positive'
```

Comment:
@evgenibir You are using this values `positive`, `negative` and `gray` in multiple places. You should have them as constants

Reply:
@Alex-IS fixed